SteamKit2/SteamKit2/Util/DebugNetworkListener.cs

```diff
@@ -63,7 +63,7 @@ public NetHookNetworkListener(ILogContext? log = null)
             );
             Directory.CreateDirectory( LogDirectory );
 
-            log.LogDebug( CategoryName, $"Created nethook directory: {LogDirectory}" );
+            this.log.LogDebug( CategoryName, $"Created nethook directory: {LogDirectory}" );

```

Comment:
what's with this one?

Reply:
Allows it to log to the default one if one isn't provided in the parameters